test/test_signal.py

```diff
@@ -0,0 +1,45 @@
+import asyncio
+import awaitwhat
+
+
+FUTURES = []
+
+
+async def main():
+    await do_work()
+
+
+async def do_work():
+    futs = [frob_a_tree() for i in range(1)]
+    await asyncio.gather(*futs)
+
+
+async def frob_a_tree():
+    await b_tree()
+
+
+async def b_tree():
+    f = asyncio.Future()
+    FUTURES.append(f)
+    await asyncio.shield(frob_a_branch(f))
+
+
+async def frob_a_branch(f):
+    await b_branch(f)
+
+
+async def b_branch(f):
+    await f
+
+
+def name(t):
+    try:
+        # Python3.8
+        return t.get_name()
+    except AttributeError:
+        return f"Task-{id(t)}"
+
+
+if __name__ == "__main__":
+    awaitwhat.utils.helpers.setup()
```

Comment:
Personally I prefer to have a more easier way of enabling `signal_handler`, e.g.: an option wrapper or something similar.

In addition, the current style doesn't really describe what kind of helpers are being configured.